.travis.yml

```diff
@@ -35,6 +35,14 @@ jobs:
           on:
             repo: open-policy-agent/gatekeeper
             tags: true # only push release image tag for tags
+    - stage: "helm build, test"
+      script:
+        - make manager
```

Comment:
We can drop the `make manager` line here, linting is taken care of elsewhere as well